src/Terrain/TerrainType.java

```diff
@@ -90,14 +90,14 @@ private TerrainType(int attributeFlags, int defense, Color mainColor, String nam
   private static final String TEMP_AIRPORT_NAME = "TEMP_AIRPORT";
   public static final TerrainType TEMP_AIRPORT = new TerrainType( TEMP_AIRPORT_FLAGS, TEMP_AIRPORT_DEFENSE, TEMP_AIRPORT_COLOR, TEMP_AIRPORT_NAME, GRASS );
 
-  private static final int BRIDGE_FLAGS = LAND | WATER;
+  private static final int BRIDGE_FLAGS = LAND;
   private static final int BRIDGE_DEFENSE = 0;
   private static final Color BRIDGE_COLOR = new Color(189, 189, 189);
   private static final String BRIDGE_NAME = "BRIDGE";
   public static final TerrainType BRIDGE = new TerrainType( BRIDGE_FLAGS, BRIDGE_DEFENSE, BRIDGE_COLOR, BRIDGE_NAME, SEA );
 
   private static final int CITY_FLAGS = LAND | CAPTURABLE | PROFITABLE | PROVIDES_COVER | HEALS_LAND;
-  private static final int CITY_DEFENSE = 2;
+  private static final int CITY_DEFENSE = 3;
```

Comment:
I had sort of assumed we were using AW2 values for things - seems out of scope for just adding a CO, but I don't object to using AW2 values.

Reply:
It is out of scope, but he was slowed by bridges and I didn't like that part.